src/commands/deploy/index.ts

```diff
@@ -109,7 +109,19 @@ Support for package.json's main field, and intrinsic index.js entrypoints are co
         'build',
       ),
     )
-    .option('--build', 'Run build command before deploying', false)
+    .addOption(
+      new Option('--build', 'Do not use - this is now the default. Will be removed in future versions.')
```

Comment:
FWIW I don't think this string can ever be seen

Reply:
Yeah these get added to generated cli.netlify.com docs

Reply:
Well, because we have `hideHelp()` it isn't even included there